axe.d.ts

```diff
@@ -71,6 +71,24 @@ declare namespace axe {
 		target: string[];
 		html: string;
 	}
+	interface RuleLocale {
+		[key: string]: {
+			description: string;
+			help: string;
+		};
+	}
+	interface CheckLocale {
+		[key: string]: {
+			pass: string;
+			fail: string;
+			incomplete: string | { [key: string]: string };
+		};
+	}
+	interface Locale {
```

Comment:
need to add `ts` tests for these changes by perhaps invoking Spec with these argument changes.